prometheus.lua

```diff
@@ -807,6 +816,8 @@ local function register(self, name, help, label_names, buckets, typ)
     -- ['my.net']['200'][LEAF_KEY] = 'http_count{host="my.net",status="200"}'
     -- ['my.net']['500'][LEAF_KEY] = 'http_count{host="my.net",status="500"}'
     lookup = {},
+    lookup_size = 0,
+    lookup_max_size = lookup_max_size or 100,
```

Comment:
Any thoughts on whether 100 will be sufficient for most users? On my personal web server I currently have ~500 metric label combinations for the `nginx_http_requests_total` metric that has two fields (server name and response code).

Given that most users will probably have a small number of metrics, but potentially large metric cardinality (at least host + status) I would probably suggest a much higher default (at least 1000).

Reply:
It is definitely not enough :slightly_smiling_face: Our servers range from 200 to about 3000 timeseries in total, and we have quite simple APIs with just a few endpoints (but a lot of various metrics).

Reply:
I agree.